04_environment/index.tex

```diff
@@ -1 +1,400 @@
+
+\subsection{Future Work Concepts}
+
+For the Future implementations, our additions focus on further exploring the dynamics of our agents and allowing for a more complex decision-making interaction between clients and the environment. Some of these design ideas were later implemented in self-organising multi-agent system and others are expressed for the sake of completeness and future work recommendations.\\
+
+Some changes have been made to make foraging more volatile and unpredictable. These will add to the complexity of the system, allowing the teams to further evaluate the clients’ performance on observing and using learned knowledge to take action.
+
+\subsubsection{Incorporated Changes: Population density}
+
+Introducing a model that controls the population density of deer allows us to increase the complexity of the foraging function. In short, the deer hunting capacity decreases with decreasing population and increases with an increasing population. This will then influence the maximum deer per hunt parameter(n), which in turn results in a more complex return calculation function (\texttt{\textbf{DtotalReturn}}), which would make it harder for agents to figure out forage tier boundaries to optimise their strategies.\\
+
+The population change can be dynamic depending on various environmental factors, fetched from the rest of the system. For example, a Disaster could supposedly cause the population density of deer to halve, making it harder to forage and therefore, causing lower returns. For future implementations, the team uses a single species population model and more specifically logistic modelling for the deer and fish population. The model can be described in the equation below.\\
+
+\begin{equation}
+\frac{\mathrm{d} P}{\mathrm{~d} t}=k(N-P)
+\end{equation}
+
+\begin{itemize}
+        \item “P” is the total Population \item “N” is the maximum deer/fish population (“carrying capacity” formally)
+        \item “k” is the growth coefficient
+        \item “t” is time
+    \end{itemize}
+
+In our implementation \textbf{“k”} and \textbf{“N”} are constants but it would be interesting to see what would happen if we were to increase the growth coefficient, causing a population growth and observing the islands’ dynamics. Will they keep on “investing” higher resources each turn, taking advantage of the deer/fish abundance, or will they stick to their original, safe and static strategy?\\
```

Comment:
go to math mode for constants:

$k$ ... $N$  -- as suggested in the guidelines!

Reply:
yepp this is v important - please use the math mode

Reply:
also there is no need to add \\ after a paragraph - we don't do this anywhere else in the document and it creates an inconsistent style (have seen this in multiple places)